common_products.sh

```diff
@@ -9,17 +9,25 @@
 # product strings: fennec, fennec-nightly, fenix-nightly, fenix-performance
 PRODUCTID=$1
 
+APPLINK_URL="https://example.com"
+
 # Define "parameters" needed to execute all the tests. These names should
 # be self explanatory. See README.md for customization information.
+#
+# fenix-nightly and fennec-nightly launch to different activities when you
+# hit the homescreen icon so the discrepancy is intentional.
+#
+# When they print logs, they both display "Fully drawn...HomeActivity" though
+# because this is ultimately the final activity displayed.
 fenix_homeactivity_start_command='am start-activity org.mozilla.fenix.nightly/org.mozilla.fenix.HomeActivity'
-fenix_applink_start_command='am start-activity -t "text/html" -d "about:blank" -a android.intent.action.VIEW org.mozilla.fenix.nightly/org.mozilla.fenix.IntentReceiverActivity'
+fenix_applink_start_command="am start-activity -t 'text/html' -d "$APPLINK_URL" -a android.intent.action.VIEW org.mozilla.fenix.nightly/org.mozilla.fenix.IntentReceiverActivity"
```

Comment:
With the quote changes, I believe this is equivalent to passing the contents of `$APPLINK_URL` without surrounding quotes. That is:

```
fenix_applink_start_command="am start-activity -t 'text/html' -d "$APPLINK_URL"...
                            ^-- start of string                  ^-- end of string
```

That's technically fine, but it would break if there's any funny business (e.g. spaces) in `$APPLINK_URL`. Maybe change the double quotes to single quotes?

Reply:
Oops 🤦‍♂